README.md

```diff
@@ -75,19 +101,24 @@ func NewConfig() *Config
 
 #### New
 
-`New` creates an instance of your module. It must return an `io.Closer` and an `error.
+`New` creates an instance of your module. It must return an `io.Closer` and an `error`.
```

Comment:
Whenever possible, when mentioning packages and interfaces, don't hesitate to link the godocs for it, for the aforementioned reason

```suggestion
`New` creates an instance of your module. It must return an [`io.Closer`](https://golang.org/pkg/io/#Closer) and an [`error`](https://golang.org/pkg/builtin/#error).
```